src/configurations/destinations/http/schema.json

```diff
@@ -27,6 +27,10 @@
         "type": "boolean",
         "default": false
       },
+      "isDefaultMapping": {
+        "type": "boolean",
+        "default": true
```

Comment:
default true?

Reply:
Yeah, default is true because currently default mapping is $ -> $
We just want to improve UX by hiding the mapping component by default and show this checkbox enabled. If this is disabled the mapping component is visible